TombEngine/Game/items.cpp

```diff
@@ -755,3 +757,47 @@ void DoDamage(ItemInfo* item, int damage)
 		}
 	}
 }
+
+void PartialItemHit(ItemInfo* itemHit, int damage, int grenade)
```

Comment:
Why this function is called `Partial`?

Reply:
It's done to avoid having same code copy-pasted

Reply:
done